docs/storage/upload/README.md

```diff
@@ -0,0 +1,20 @@
+## upload sample
+
+This sample is a snippet of code with which you can create a CLI
+that allows you to upload a file to your Google Cloud Storage bucket.
+
+### Installing it
+```shell
+go get github.com/GoogleCloudPlatform/golang-samples/docs/storage/upload
```

Comment:
This package needs to live in github.com/GoogleCloudPlatform/golang-samples/storage instead. I don't know what docs subdirectory is. @broady?